readme.md

```diff
@@ -63,6 +65,35 @@ console.log(dargs(input, {includes: includes}));
 	'--sad=:(''
 ]
 */
+
+
+console.log(dargs(input, {shortForms: shortForms}));
```

Comment:
To make it clearer what this does I think we should drop using the `input` fixture and just have one inline input here.